### PR TITLE
Using concurrently to run npm scripts concurrently...!

### PR DESCRIPTION
**Pull request by DamianMullins**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,11 +49,11 @@
     "exports": false
   },
   "scripts": {
-    "prepare": "yarn run lint && yarn test && yarn run compile",
-    "compile": "babel -d dist src",
+    "prepare": "concurrently -n \"lint,compile,test\" -c \"blue,yellow,green\" --kill-others-on-fail \"yarn lint\" \"yarn compile\" \"yarn run test\"",
```
Comment (ashleynolan) [inline package.json]: Instead of escaping these, can they not just be using single quotes – or are double quotes needed for this?

Comment (DamianMullins) [reply package.json]: I'm not sure, I'll test it out 😸

Comment (DamianMullins) [reply package.json]: Works with single quotes 😸  I also used the shortened argument names where possible in my latest code push.

Comment (ashleynolan) [reply package.json]: 👍  💯

**Pull request merged**